base/xx-cc

```diff
@@ -8,15 +8,28 @@ if [ -n "$XX_MIRROR" ]; then
   mirrors="$XX_MIRROR"
 fi
 
+_wget() {
+  if ! wget "$@" && "$(wget --help 2>&1 | head -n1 | cut -d" " -f1)" != "GNU"; then
```

Comment:
if wget returns an error but it is not gnu then this function will return without an error atm

Reply:
Yes indeed. I have created 2 commits, the first one that keeps the wget logic in xx-cc and another one that adds `xx-wget` for our purpose. Let me know what suits better.